e2e/nomostest/nt.go

```diff
@@ -615,19 +609,18 @@ func (nt *NT) portForwardGitServer() {
 		// re-init all repos
 		InitGitRepos(nt, allRepos...)
 		// attempt to recover by re-pushing the local repo states
-		for nn, remoteRepo := range nt.RemoteRepositories {
-			remoteURL := nt.GitProvider.RemoteURL(newPort, nn.String())
-			remoteRepo.pushAllToRemote(remoteURL)
-			remoteRepo.RemoteURL = remoteURL
+		for _, remoteRepo := range nt.RemoteRepositories {
+			remoteRepo.pushAllToRemote()
```

Comment:
Kinda surprised this isn't public yet. Is it because Repository is still in this package?

Maybe that's another PR to pull out.

Reply:
Yeah it's still in this package and the method is only used within the package currently. The method will need to be public if it's moved to its own package